conbench/tests/populate_local_conbench.py

```diff
@@ -14,8 +14,13 @@ def generate_benchmarks_data(
     benchmark_language,
     timestamp,
     hardware_type,
+    is_nightly,
```

Comment:
I couldn't find an appropriate place to add a unit test for this behavior, because it looks like we don't unit test how the HTML displays. So I modified this script and checked that the behavior works as expected locally.